src/be_db_gateway_status.erl

```diff
@@ -275,7 +275,7 @@ peer_gps_fix_quality(Address, PeerBook) ->
 peer_height(Address, PeerBook) ->
     case peer_metadata(<<"height">>, Address, PeerBook) of
         undefined -> undefined;
-        Height when is_integer(Height) and Height > 0 -> Height;
```

Comment:
you wanted `andalso` here.  it's `and` that has the funny rules, not guard context.